test/e2eTests/translator.js

```diff
@@ -0,0 +1,30 @@
+const I18n = require('./i18n');
+
+// Create the translator as a singleton to share the instance across the app
+// Usage: declare it and use the translate function
+// const t = require('./../translator')
+// t.translate(key)
+const translator = (function () {
```

Comment:
Check [this](http://www.adam-bien.com/roller/abien/entry/singleton_pattern_in_es6_and) out. I think it's a cleaner way to create a singleton that uses ES6 syntax. What do you think?

Reply:
Will check and include for next refactoring. Thanks!